docs/components/query-frontend.md

```diff
@@ -47,6 +47,8 @@ Query Frontend supports caching query results and reuses them on subsequent quer
 #### In-memory
 
 ```yaml mdox-exec="go run scripts/cfggen/main.go --name=queryfrontend.InMemoryResponseCacheConfig"
+# command-line-arguments
+ld: warning: ignoring duplicate libraries: '-lproc'
```

Comment:
Something went wrong here? :thinking: maybe you could try regenerating?

Reply:
make docs is generating this, fixed the PR and CI is green now